style/battle.css

```diff
@@ -630,7 +630,7 @@ License: GPLv2
 	background: #333333;
 	color: #CCCCCC;
 }
-.dark .battle-log .chat strong,
+.dark .battle-log .chat > strong,
```

Comment:
Oh, you definitely don't want to use `>` for that, that's incredibly brittle. (Brittleness is the main reason I don't like `>`)